cmd/kube-controller-manager/app/batch.go

```diff
@@ -41,14 +39,13 @@ func startJobController(ctx ControllerContext) (bool, error) {
 }
 
 func startCronJobController(ctx ControllerContext) (bool, error) {
-	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "batch", Version: "v2alpha1", Resource: "cronjobs"}] {
+	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "batch", Version: "v2alpha1", Resource: "cronjobs"}] &&
+		!ctx.AvailableResources[schema.GroupVersionResource{Group: "batch", Version: "v1beta1", Resource: "cronjobs"}] {
```

Comment:
`batch/v1beta1` cronjob clientsets are used in cronjob controller code, so `batch/v1beta1` has to be enabled?

Reply:
Yeah, I initially thought that maybe be possible to work with both, but it's not. I forgot to update it. Will do.